api/core/model_runtime/model_providers/bedrock/llm/llm.py

```diff
@@ -410,12 +412,12 @@ def _handle_generate_stream_response(self, model: str, credentials: dict, respon
             else:
                 raise ValueError(f"Got unknown model prefix {model_prefix} when handling stream response")
 
-            index += 1
-           
+            # transform assistant message to prompt message
             assistant_prompt_message = AssistantPromptMessage(
-                content = content_delta if content_delta else '',
+                content=content_delta.get("completion") if content_delta else '',
```

Comment:
This is specific to anthropic's return format. Could you change line `401` to `content_delta = payload.get("completion") instead

```suggestion
                content=content_delta if content_delta else '',
```

Reply:
> This is specific to anthropic's return format. Could you change line `401` to `content_delta = payload.get("completion") instead

Sure, I also found this issue ,and resolved it at forked branch, so i will send the other PR to resolve it.

Reply:
> content=content_delta.get("completion") if content_delta else '',

it should be content=content_delta.get("completion") if content_delta else '', because for bedrock ,it is special format.